Firmware/MotorControl/motor.cpp

```diff
@@ -66,6 +70,9 @@ bool Motor::apply_config() {
     config_.parent = this;
     is_calibrated_ = config_.pre_calibrated;
     update_current_controller_gains();
+    fet_thermistor_.motor_ = this;
+    motor_thermistor_.motor_ = this;
+    motor_thermistor_.apply_config();
```

Comment:
While this is a valid place for this call I would probably rather move it to the top level (`config_apply_all` in main.cpp). The reason is mainly consistency: the related config operations (load, store, clear) are all done directly from the top level too (also for the Motor object, which, to the user, is a subobject of Axis).

Reply:
Addressed in fd848f2